portal/src/App.tsx

```diff
@@ -14,7 +16,48 @@ const query = graphql`
 const ShowQueryResult: React.FC<AppQueryResponse> = function ShowQueryResult(
   props: AppQueryResponse
 ) {
-  return <div>{props.viewer?.id}</div>;
+  const { viewer } = props;
+  const onClickLogout = useCallback(() => {
+    authgear
+      .logout({
+        redirectURI: window.location.href,
```

Comment:
Since we're doing exact match validation of redirectURI at server-side, I suppose we should give a hard-coded value here (i.e. through build time variables), since the current page URL may change.

Reply:
Sure. I am going to change it to `window.location.origin + "/"`.